### ISSUE TITLE
False positive 932250 when sending "sh" parameter

### ISSUE THREAD

**Issue by S0obi** [coreruleset/coreruleset#4241]
### Description

Keywords "sh" or "zsh" are currently blocked by 932250. I think it's too strict for PL1, in our specific case, this is a search form and users can search for "sh" (beginning of many english words).

### How to reproduce the misbehavior (-> curl call)

```bash
curl -H "x-format-output: txt-matched-rules" https://sandbox.coreruleset.org/\?searchValue\=sh  
932250 PL1 Remote Command Execution: Direct Unix Command Execution
949110 PL1 Inbound Anomaly Score Exceeded (Total Score: 5)
```

### Logs

```
ModSecurity: Warning. Matched "Operator `Rx' with parameter `(?i)(?:^|b[\"'\)\[\x5c]*(?:(?:(?:\|\||&&)[\s\x0b]*)?\$[!#\(\*\-0-9\?@_a-\{]*)?\x5c?u[\"'\)\[\x5c]*(?:(?:(?:\|\||&&)[\s\x0b]*)?\$[!#\(\*\-0-9\?@_a-\{]*)?\x5c?s[\"'\)\[\x5c]*(?:(?:(?:\|\||&&)[\s\x0b]*)? (7054 characters omitted)' against variable `ARGS:searchValue' (Value: `sh' ) [file "/etc/nginx/modsec/coreruleset-4.17.1/rules/REQUEST-932-APPLICATION-ATTACK-RCE.conf"] [line "495"] [id "932250"] [rev ""] [msg "Remote Command Execution: Direct Unix Command Execution"] [data "Matched Data: sh found within ARGS:searchValue: sh"] [severity "2"] [ver "OWASP_CRS/4.17.1"] [maturity "0"] [accuracy "0"] [tag "application-multi"] [tag "language-shell"] [tag "platform-unix"] [tag "attack-rce"] [tag "paranoia-level/1"] [tag "OWASP_CRS"] [tag "OWASP_CRS/ATTACK-RCE"] [tag "capec/1000/152/248/88"] [hostname "example.com"] [uri "/inspection/users"] [unique_id "175507619660.834755"] [ref "o0,2v34,2"]
ModSecurity: Access denied with code 403 (phase 2). Matched "Operator `Ge' with parameter `5' against variable `TX:BLOCKING_INBOUND_ANOMALY_SCORE' (Value: `5' ) [file "/etc/nginx/modsec/coreruleset-4.17.1/rules/REQUEST-949-BLOCKING-EVALUATION.conf"] [line "222"] [id "949110"] [rev ""] [msg "Inbound Anomaly Score Exceeded (Total Score: 5)"] [data ""] [severity "0"] [ver "OWASP_CRS/4.17.1"] [maturity "0"] [accuracy "0"] [tag "anomaly-evaluation"] [tag "OWASP_CRS"] [hostname "example.com"] [uri "/inspection/users"] [unique_id "175507619660.834755"] [ref ""]
```

### Your Environment

* CRS version : 4.17.1
* Paranoia level setting : PL1
* ModSecurity version : 3.0.14
* Web Server : nginx

### Confirmation

[X] I have removed any personal data (email addresses, IP addresses, passwords, domain names) from any logs posted.

**Comment by EsadCetiner** [comment]
@S0obi Thanks for the report, we'll have to create a new rule that handles blocking the execution of commands with no arguments. This only really needs to be done for a handful of commands.

**Comment by EsadCetiner** [comment]
@S0obi I opened a PR which should resolve similar issues like this in #4273 

False positives like this should be much rarer now since it's now only targeting the handful of commands that can be meaningfully executed with no arguments.
